app/views/events/_form.html.erb

```diff
@@ -29,95 +40,90 @@
           <%= f.radio_button 'online', 'true', class: 'location', checked: @event.online?, autocomplete: 'off' %>
           <i class="fa fa-desktop" aria-hidden="true" id="online-symbol"></i> Online
         </label>
+        <label class="btn btn-default <%= 'active' unless @event.online? %>">
+          <%= f.radio_button 'online', 'false', class: 'location', checked: !@event.online?, autocomplete: 'off' %>
+          <%= icon_for(:face_to_face, :small)%> Hybrid
+        </label>
       </div>
     </div>
 
-    <%# Necessary to allow removal of all field locks %>
-    <%= hidden_field_tag 'event[locked_fields][]', '' %>
-
-    <%= f.input :title, as: :string, field_lock: true %>
-
-    <%= f.input :subtitle, field_lock: true %>
-
-    <%= render partial: 'common/url_checker', locals: { f: f, url: events_check_exists_path } %>
-
-    <%= f.input :start, as: :datetime_picker, field_lock: true %>
-
-    <%= f.input :end, as: :datetime_picker, field_lock: true %>
-
     <div class="address_content">
       <%= render partial: 'events/partials/address_finder', locals: { f: f } %>
     </div>
 
-    <%= f.input :organizer, field_lock: true %>
+    <%= f.multi_input :keywords %>
 
     <%= f.input :content_provider_id, label: 'Content provider (where the event metadata is obtained from)',
                 collection: ContentProvider.all, label_method: :title, value_method: :id, include_blank: true,
                 field_lock: true %>
 
-    <%= f.dropdown :node_ids, options: format_for_dropdown(Node.order(:name).all), label: 'Nodes' %>
+    <%= collapsible_panel('Optional Settings', 'optional_settings') do %>
```

Comment:
perhaps you could call it 'Extra settings'? Since only the ones with a star are optional, strictly speaking